types.go

```diff
@@ -353,7 +353,7 @@ func (rr *X25) String() string {
 type RT struct {
 	Hdr        RR_Header
 	Preference uint16
-	Host       string `dns:"cdomain-name"`
+	Host       string `dns:"domain-name"`
```

Comment:
This probably should be documented either on this field or on the RR.

Something like:
```
// RFC 3597 prohibits compressing records not defined in RFC 1035.
```

Reply:
@miekg Thoughts/

Reply:
Yeah, just leaving the comment on the Host field is fine.

Reply:
@miekg Done.